lib/senders/email.js

```diff
@@ -589,6 +590,42 @@ module.exports = function (log, config, oauthdb) {
     }));
   };
 
+
+  verificationReminders.keys.forEach(key => {
+    const template = `verificationReminder${key[0].toUpperCase()}${key.substr(1)}Email`;
```

Comment:
I like this pattern in general, however it doesn't make it easy to know what the name of the email template is. I think it might be helpful to add a comment above describing them. It might also make it easier to search for in code.